src/yara.cc

```diff
@@ -519,7 +520,11 @@ void compileCallback(int error_level, const char* file_name, int line_number,
 	CompileArgs* args = (CompileArgs*) user_data;
 
 	std::ostringstream oss;
-	oss << args->rule_config->index << ":" << line_number << ":" << message << ":" << file_name;
+	oss << args->rule_config->index << ":" << line_number << ":" << message << ":";
+
+	if (file_name != NULL) {
+		oss << file_name;
+	}
```

Comment:
@miguelxpn - should we backport these fixes to the `master` branch?

Reply:
I just cherry picked them into master!

Reply:
Perfect, thanks!